server/services/caseloadService.ts

```diff
@@ -78,12 +78,15 @@ export default class CaseloadService {
         // TODO: This .then() block is temporary to reduce the number of offenders being searched in probation-offender-search. A more permanent
         //  fix will be to have an endpoint to search nomis by CRD (CVSL-492).
         //  Temporary fix is to filter only prisoners who have a CRD in the next 13 weeks before searching them on delius
-        return caseload
-          .filter(offender => offender.conditionalReleaseDate)
-          .filter(offender => moment(offender.conditionalReleaseDate, 'YYYY-MM-DD').isSameOrAfter(moment(), 'day'))
-          .filter(offender =>
-            moment(offender.conditionalReleaseDate, 'YYYY-MM-DD').isBefore(moment().add(13, 'weeks'), 'day')
-          )
+        return caseload.filter(
+          offender =>
+            (offender.conditionalReleaseDate &&
+              moment(offender.conditionalReleaseDate, 'YYYY-MM-DD').isSameOrAfter(moment(), 'day') &&
+              moment(offender.conditionalReleaseDate, 'YYYY-MM-DD').isBefore(moment().add(13, 'weeks'), 'day')) ||
+            (offender.confirmedReleaseDate &&
+              moment(offender.confirmedReleaseDate, 'YYYY-MM-DD').isSameOrAfter(moment(), 'day') &&
+              moment(offender.confirmedReleaseDate, 'YYYY-MM-DD').isBefore(moment().add(13, 'weeks'), 'day'))
+        )
```

Comment:
Instead of having to check both dates everywhere, I am thinking of putting a `releaseDate` field on the `ManagedCase` type which is a composite of the various dates (i.e. the most relevent one). This should simplify the code a bit. Its a future improvement though